examples/api-backends/gorilla-mux/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/gorilla/csrf"
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+)
+
+func main() {
+	router := mux.NewRouter()
+
+	loggingMiddleware := func(h http.Handler) http.Handler {
+		return handlers.LoggingHandler(os.Stdout, h)
+	}
+	router.Use(loggingMiddleware)
+
+	CSRFMiddleware := csrf.Protect(
+		[]byte("place-your-32-byte-long-key-here"),
```

Comment:
hoist the literal value out of the code, into a constant?

Reply:
definitely open to this; in my own csrf examples repo I have everything factored out to be configurable through cli flags and config files.

For these I tried to follow the style of the existing docs and keep it as short as possible.
It seems like in general, example/quickstart docs tend to opt for hardcoding